docs/util.md

```diff
@@ -483,7 +486,10 @@ In Java: Use `BigDecimal` to represent this value.
 
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
-| value | [double](#double) |  | Value in the range [0,1], inclusive |
+| rate | [string](#string) |  | Expects digits after a `.`, no + sign. Ex: 1.0, -1.0, 1, -1, 0.01. |
+| value | [double](#double) |  | **Deprecated.** deprecated
```

Comment:
If we deprecate a field, can we provide what field to use instead?

Reply:
The docs are generated using a script in the root of the repo, so I'd hesitate to alter the output from what it generates since future contributors might end up reverting it. That said, the comments in the code should get updated to indicate what the new field is, but you can see that the descriptions generated for deprecated fields are bugged and not outputting on the same line. I'll revert the changes generated from #15 and see about fixing all the instances in a single future PR.